Resources/translations/CmfTreeBrowserBundle.en.xlf

```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="utf-8"?>
+<xliff xmlns="urn:oasis:names:tc:xliff:document:1.2" version="1.2">
+  <file source-language="en" target-language="en" datatype="plaintext" original="file.ext">
+    <header>
+      <tool tool-id="symfony" tool-name="Symfony"/>
+    </header>
+    <body>
+      <trans-unit id="57107163072f788ccde704eedc055bbd" resname="Pick">
+        <source>Pick</source>
```

Comment:
lets use descriptive keys like in the other cmf bundles, not english words. if we ever go for Choose in english, we would translate Pick to Choose which would be weird.